django/contrib/auth/models.py

```diff
@@ -145,9 +145,9 @@ def _create_user(self, username, email, password, **extra_fields):
         if not username:
             raise ValueError('The given username must be set')
         email = self.normalize_email(email)
-        username = self.normalize_username(username)
         user = self.model(username=username, email=email, **extra_fields)
         user.set_password(password)
+        user.clean()
```

Comment:
As `clean()` can also raise `ValidationError` if overridden I would avoid calling it here and stick to calling `normalize_username()`.